rust_dev_preview/examples/aurora/src/aurora_scenario/tests.rs

```diff
@@ -923,7 +923,7 @@ async fn test_scenario_clean_up_errors() {
                     ErrorKind::Other,
                     "describe db clusters error",
                 ))),
-                HttpResponse::new(SdkBody::empty()),
+                Response::new(StatusCode::try_from(400).unwrap(), SdkBody::empty()),
```

Comment:
we should add a const constructor for status code I think.

Reply:
Will need to happen in a future release.